tests/components/air_quality/test_air_quality.py

```diff
@@ -34,3 +38,6 @@ async def test_attributes(hass):
     assert data.get(ATTR_N2O) is None
     assert data.get(ATTR_OZONE) is None
     assert data.get(ATTR_ATTRIBUTION) == "Powered by Home Assistant"
+    assert (
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5358993)